File: Beecrowd/Go/StaticPassword_1114.go
package main

import (
	"fmt"
)

func main() {
	var password int

	// The cycle repeats as long as the password is not 2002
	for password != 2002 {
		// fmt.Scan reads integers directly from the console (like sc.nextInt())
		fmt.Scan(&password)

		if password != 2002 {
			fmt.Println("Senha Invalida")
		}
	}

	fmt.Println("Acesso Permitido")
}

File: Beecrowd/Java/src/beecrowd/StaticPassword_1114.java
/*
  Problem: Static Password
  ID: 1114
*/
package beecrowd;

import java.io.IOException;
import java.util.Scanner;

public class StaticPassword_1114 {
 
    public static void main(String[] args) throws IOException {
 
		Scanner sc = new Scanner(System.in);
		
		int password = 0;
		
		while(password != 2002){
		    password = sc.nextInt();
		    
		    if(password != 2002){
		        System.out.println("Senha Invalida");
		    }
		}
		
		System.out.println("Acesso Permitido");
 
    }
 
}
